### ISSUE TITLE
sycl_cts/test_spec_constants fails on win gen12 L0

### ISSUE THREAD

**Issue by KornevNikita** [intel/llvm#19625]
### Describe the bug

See https://github.com/intel/llvm/actions/runs/16586003151/job/46913174494
```
  D:\github\_work\llvm\llvm\khronos_sycl_cts\util\logger.cpp(41): warning:
    Exception fired for case of using same kernel bundle with bundle_state::input
    state, 0 write(s) and 1 read(s) for type sycl::half
  
  D:\github\_work\llvm\llvm\khronos_sycl_cts\util\sycl_exceptions.h(184): warning:
    SYCL exception
    with category name: 'sycl'
    with code: 'invalid'
    with code raw value: '8'
    with code message: 'SYCL Error'
    with what: 'Kernel is incompatible with all devices in devs'
  
  D:\github\_work\llvm\llvm\khronos_sycl_cts\util\test_base.cpp(28): FAILED:
  explicitly with message:
    a SYCL exception was caught: Kernel is incompatible with all devices in devs
```
Looks like https://github.com/intel/llvm/commit/a50acd0244276fb9efb231abae5ce9d71495768b is guilty. @againull could you please check.

### To reproduce

1. Include a code snippet that is as short as possible
2. Specify the command which should be used to compile the program
3. Specify the command which should be used to launch the program
4. Indicate what is wrong and what was expected

### Environment

- OS: [e.g Windows/Linux]
- Target device and vendor: [e.g. Intel GPU]
- DPC++ version: [e.g. commit hash or output of `clang++ --version`]
- Dependencies version: [e.g. the output of `sycl-ls --verbose`]

**Comment by KornevNikita** [comment]
Not sure why, but it passes now. Closing.

**Comment by KornevNikita** [comment]
The issue is still here.

**Comment by AlexeySachkov** [comment]
I wonder if this is the issue that's being addressed in KhronosGroup/SYCL-CTS#1144

**Comment by KornevNikita** [comment]
> I wonder if this is the issue that's being addressed in [KhronosGroup/SYCL-CTS#1144](https://github.com/KhronosGroup/SYCL-CTS/pull/1144)

yeah, looks related. I'll try to re-enable if after 1144 is merged.

**Comment by AlexeySachkov** [comment]
@KornevNikita, I supposed that we can close this one already, right?

**Comment by KornevNikita** [comment]
Yep, looks like it doesn't fail anymore. Closing.
